_posts/2018-07-30-ARIA-states.md

```diff
@@ -0,0 +1,29 @@
+---
+layout: post
+title: ARIA States
+description: A quick introduction to ARIA's state system.
+author: Alex Brenon
+categories: basics
+published: true
+---
+# Aria States
+[ARIA defines multiple "states"](https://www.w3.org/WAI/PF/aria-1.1/states_and_properties) that can be applied to elements. States are likely to change as the user interacts with the page. States are inserted into elements as follows: `<element aria-state="value"></element>` . Example: 
+```html
+<span aria-busy="true">This element is currently being updated. </span> 
+```
+There are 9 states that can be used in ARIA. They fall into 3 categories as defined by W3. The categories are Widget Attributes - states that give information about user input elements; Live region attributes - states that are specific to regions that will update while the page is being viewed; and Drag-and-Drop attributes - states of drag-and-drop input elements.
```

Comment:
W3 => W3C

Also, same comment as in previous article, it'd be nice to see this paragraph listed out instead.